src/fetch_foursquare_info.py

```diff
@@ -0,0 +1,129 @@
+import os.path
+import requests
+import numpy as np
+import pandas as pd
+import re
+import json
+import datetime
+
+# Setting dataset paths
+REIMBURSEMENTS_DATASET_PATH = os.path.join('data', '2016-11-22-reimbursements.xz')
+COMPANIES_DATASET_PATH = os.path.join('data', '2016-09-03-companies.xz')
+FOURSQUARE_DATASET_PATH = os.path.join('data', 'foursquare-companies.json')
+
+# These are the keys to access Foursquare's API. You can create your own through the link bellow
+# https://pt.foursquare.com/developers/register
+CLIENT_ID = ''
+CLIENT_SECRET = ''
+# This is in YYYYMMDD format.
+VERSION = '20161021'
+
+# Required params to make a request to Foursquare's API
+DEFAULT_PARAMS = {'client_id': CLIENT_ID,
+                'client_secret': CLIENT_SECRET,
+                'v': VERSION}
+
+def load_cnpjs(subquota_description):
+    u_cols = ['cnpj_cpf', 'subquota_description']
+    docs = pd.read_csv(REIMBURSEMENTS_DATASET_PATH,
+                        low_memory=False,
+                        usecols=u_cols,
+                        dtype={'cnpj_cpf': np.str})
+    meals = docs[docs.subquota_description == subquota_description]
+    return meals['cnpj_cpf'].unique()
+
+def load_companies():
+    u_cols = ['cnpj', 'trade_name', 'zip_code', 'latitude', 'longitude']
+    all_companies = pd.read_csv(COMPANIES_DATASET_PATH,
+                                low_memory=False,
+                                usecols=u_cols,
+                                dtype={'trade_name': np.str})
+    all_companies['clean_cnpj'] = all_companies['cnpj'].map(cleanup_cnpj)
+    relevant_cnpjs = load_cnpjs('Congressperson meal')
+    return all_companies[all_companies['clean_cnpj'].isin(relevant_cnpjs)]
+
+def cleanup_cnpj(cnpj):
+    return re.sub("\D", "", cnpj)
+
+def remaining_companies(companies):
+    fetched_companies = load_foursquare_companies_dataset()
+    return companies[~companies['cnpj'].isin(fetched_companies['cnpj'])]
+
+def load_foursquare_companies_dataset():
+    if os.path.exists(FOURSQUARE_DATASET_PATH):
+        return pd.read_json(FOURSQUARE_DATASET_PATH)
+    else:
+        return pd.DataFrame(columns=['cnpj'])
+
+def get_venue(company):
```

Comment:
What kind of data is `company`? A Pandas row? A CNPJ string? A parsed JSON object? I'd suggest adding [`__docstrings__`](https://www.python.org/dev/peps/pep-0257/) to (almost) all functions to make your logic clearer…